examples/cis2-wccd/src/lib.rs

```diff
@@ -65,15 +79,25 @@ struct AddressState<S> {
 #[derive(Serial, DeserialWithState, StateClone)]
 #[concordium(state_parameter = "S")]
 struct State<S> {
-    /// The state the one token.
+    /// The admin address can upgrade the contract, pause and unpause the
+    /// contract, transfer the admin address to a new address, set
+    /// implementors, and update the metadata URL in the contract.
+    admin:        Address,
+    /// Contract is paused if `paused = true` and unpaused if `paused = false`.
+    paused:       bool,
+    /// Map specifying the `AddressState` (balance, and operators) for every
+    /// address.
     token:        StateMap<Address, AddressState<S>, S>,
     /// Map with contract addresses providing implementations of additional
     /// standards.
     implementors: StateMap<StandardIdentifierOwned, Vec<ContractAddress>, S>,
+    /// The metadata URL for the wCCD token following the specification RFC1738.
+    #[concordium(size_length = 2)]
+    url:          String,
```

Comment:
Should this be a StateBox wrapped? I guess we don't need this field apart from some view functions, and the URL is generally going to be more than 8 bytes.

Reply:
yes, I moved it into the StateBox. I also removed the `#[concordium(size_length = 2)]` because the length doesn't matter that much anymore.